nautobot/core/tests/test_views.py

```diff
@@ -215,6 +219,38 @@ def test_filtering_crafted_query_params(self):
             response_content,
         )
 
+    def test_all_list_views_have_set_filterset_form(self):
+        """Test that all ListView classes in Nautobot apps have a non-None `filterset_form` attribute."""
+        listview_classes = []
+
+        # Filter only 'nautobot' apps
+        nautobot_apps = [app for app in settings.INSTALLED_APPS if app.startswith("nautobot")]
+
+        nautobot_apps = sorted(set(nautobot_apps))
+
+        for app in nautobot_apps:
+            try:
+                views_module = importlib.import_module(f"{app}.views")
+            except ModuleNotFoundError:
+                continue
+            for name, obj in inspect.getmembers(views_module, inspect.isclass):
+                if name.endswith("ListView"):
```

Comment:
1) Can we check for `issubclass(ObjectListView)` instead?
2) Probably should also check for UIViewSet `issubclass(ObjectListViewMixin)` as well.

As an alternative approach, I wonder if it'd be better add a new test to the generic `ListObjectsViewTestCase` instead of having one test case that tries to handle every view at once - thoughts?